xrootd/client/filesystem_test.go

```diff
@@ -62,6 +62,7 @@ func testFileSystem_Open(t *testing.T, addr string, options xrdfs.OpenOptions, w
 	if err != nil {
 		t.Fatalf("invalid open call: %v", err)
 	}
+	defer gotFile.Close(context.Background())
```

Comment:
yep. but I think we are also missing a call to `Close` in https://github.com/EgorMatirov/hep/blob/b01fecaacad6396e62701a8de9eb97c3fbdbb311/xrootd/client/filesystem_mock_test.go#L243

Reply:
That's mocked test. Server exists in the other goroutine and doesn't actually allocate any file handlers. I am not sure that we need to bother with that workflow here (it would probably require to wait for another request from the server side).

Since there is `TestFile_Close_Mock` we still test that we can (un)marshal and parse `Close` from both the server and the client side.

Reply:
ok.

Reply:
perhaps just a little `TODO/FIXME` note there?

Reply:
thanks :)